### PR TITLE
PCC02 d3prof3t

### PR DESCRIPTION
**Pull request by d3prof3t**
Difficulty level (1-10): [3]
Estimated time spent (hours): [4]
Completed (yes/no): [Yes]
I stretched my coding skills (if yes what did you learn?): [some itertools magic]
Other feedback (what can we improve?): []

### CODE REVIEW

File: 02/d3prof3t/game.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+def draw_letters():
+    """Pick NUM_LETTERS letters randomly. Hint: use stdlib random"""
+    return random.sample(POUCH, NUM_LETTERS)
+
+def input_word(draw):
+    """Ask player for a word and validate against draw.
+    Use _validation(word, draw) helper."""
+    word = input()
+    if word:
+        if _validation(word, draw):
```
Comment (pybites) [inline 02/d3prof3t/game.py]: Maybe better: try _validation exception ValueErrror return None?

**Comment by pybites** [comment]
We love itertools! Nice job

File: 02/d3prof3t/game.py (hunk 1 above)
Comment (d3prof3t) [reply 02/d3prof3t/game.py]: Thanks for the review!
This seems to be a good pythonic way to do it. Will update and send the PR right away

Comment (d3prof3t) [reply 02/d3prof3t/game.py]: PR >> https://github.com/pybites/challenges/pull/234

**Pull request merged**